WcaOnRails/app/models/continent.rb

```diff
@@ -11,11 +11,11 @@ def self.country_ids(continent_id)
   end
 
   def name
-    I18n.t(recordName, scope: :continents)
+    I18n.t(read_attribute(:name), scope: :continents)
```

Comment:
I changed the continent name lookups to lookup by the continent
name, rather than the recordName, because looking up by the record name
just feels weird to me. @viroulep, it looks like you set this up to lookup by recordName in 5bf5a5514. Any idea why you chose to do it that way? Perhaps there's something wrong with what I've done here that I'm not realizing?